Azure_Pipelines/deploy_all_new.yaml

```
@@ -0,0 +1,11 @@
+name: Deploy_All
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)